### DISCUSSION TITLE
Adding BTC PAY as payment processor?

### DISCUSSION THREAD

**Discussion by iraszl** [pay-rails/pay#420] [Ideas]
[BTC PAY](https://github.com/btcpayserver/btcpayserver) is an open source payment system that can process Bitcoin and Bitcoin Lightning Network payments. It's the largest and most supported project.

Would it be acceptable or interesting to integrate it with the pay gem?

**Comment by excid3** [comment]
This might make for a great plugin. I don't have enough time to officially support this in the library, but it's also the perfect thing we could build as a plugin.

The way it works right now is we look for classes that are defined so it shouldn't take many changes to support a plugin that adds BTC Pay.
